File: 240204/코드네임/code-name.java
import java.io.*;
import java.util.*;

class Agent {
    String codeName;
    int point;
    public Agent(String codeName, int point) {
        this.codeName = codeName;
        this.point = point;
    }

    public void printAgentInfo() {
        System.out.println(String.format("%s %d", this.codeName, this.point));
    }
}

public class Main {
    public static int length = 5;
    public static List<Agent> agents = new ArrayList<>();

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        for (int i = 0; i < length; i++) {
            String[] input = br.readLine().split(" ");
            agents.add(new Agent(input[0], Integer.parseInt(input[1])));
        }
        for (int i = 0; i < length; i++) {
            int max = agents.get(i).point;
            for (int j = i + 1; j < length; j++) {
                if (agents.get(j).point < max) {
                    Agent temp = agents.get(j);
                    agents.set(j, agents.get(i));
                    agents.set(i, temp);
                }
            }
        }
        agents.get(0).printAgentInfo();
    }
}

File: 240317/코드네임/code-name.py
class Agent:
    def __init__(self, codename, score):
        self.codename = codename
        self.score = score

agents = []
for _ in range(5):
    codename, score = input().split()
    agents.append(Agent(codename, int(score)))

min_val = 100
min_agent = None

for ele in agents:
    if ele.score < min_val:
        min_agent = ele
        min_val = min_agent.score

print(min_agent.codename, min_agent.score)